File: phase-3/procedural.js
let a = 0;

if (a === 0) {
    console.log('neither odd or even')
} else if (a % 2 === 0) {
    console.log('even')
} else {
    console.log('odd')
}

switch (a % 2) {
    case 0:
        console.log('even')
    default:
        console.log('odd')
}

File: phase-3/procedural.rb
a = 100
if a == 0
  puts 'neither odd or even'
elsif a.even?
  puts 'even'
else
  puts 'odd'
end

case true
when a === 0
  puts 'neither'

when a % 2 === 0
  puts 'even'
when a.odd?
  puts 'odd'
end

def greet(title, name)
  puts "Hello, #{title} #{name}"
end

greet('Mr', 'Bob')
greet('Bob', 'Mr')

def greet(title: 'Mr', name: '')
  puts "Hello, #{title} #{name}"
end

greet(title: 'Mr', name: 'Bob')

num = 1

puts num -= 1 while num > 0
